src/pretix/plugins/checkinlists/exporters.py

```diff
@@ -382,8 +382,9 @@ def get_story(self, doc, form_data):
                 iac = ""
 
             name = op.attendee_name or (op.addon_to.attendee_name if op.addon_to else '') or ian
-            if iac:
-                name += "<br/>" + iac
+            company = op.company or (op.addon_to.company if op.addon_to else '') or iac
+            if company:
+                name += "<br/>" + company
```

Comment:
It was like this before, but maybe it is worth a look: this generates an empty line if no name is given. It looks a bit strange in the PDF. Not sure how often that happens, that e.g. a company orders tickets, but does not provide individual names.
```suggestion
                if name:
                    name += "<br>"
                name += company
```